packages/hash/shared/src/types.ts

```diff
@@ -1,9 +1,8 @@
 import { BaseUri, VersionedUri } from "@blockprotocol/type-system-web";
 import slugify from "slugify";
+import { SYSTEM_ACCOUNT_SHORTNAME } from "@hashintel/hash-backend-utils/system";
```

Comment:
This import makes frontend depend on backend-utils through imports, which we didn't have before. This means the frontend dockerfile doesn't build anymore for the production docker-compose. I suggest we either move these system env variables into the shared package.

Even if we add backend-utils to the dockerfile, there are other compilation errors becuase of `getRequiredEnv`:
```
#0 12.47 ../backend-utils/src/environment.ts
#0 12.47 Module parse failed: Unexpected token (15:35)
#0 12.47 You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
#0 12.47 |  * @todo Replace with https://www.npmjs.com/package/envalid
#0 12.47 |  */
#0 12.47 > export const getRequiredEnv = (name: string) => {
#0 12.47 |   const value = process.env[name];
#0 12.47 |   if (value === undefined) {
#0 12.47
#0 12.47 Import trace for requested module:
#0 12.47 ../backend-utils/src/environment.ts
#0 12.47 ../backend-utils/src/system.ts
#0 12.47 ../shared/src/types.ts
#0 12.47
#0 12.47
#0 12.47 > Build failed because of webpack errors
#0 12.51 error Command failed with exit code 1.
#0 12.51 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Note that this error may be because of the `@hashintel/hash-backend-utils/system` package not being in package.json. Before this error, we are unable to resolve the path without adding the following to `packages/hash/docker/frontend/prod/Dockerfile`:
```
COPY packages/hash/backend-utils/package.json packages/hash/backend-utils/

COPY packages/hash/backend-utils packages/hash/backend-utils
```